graylog2-server/src/main/java/org/graylog2/plugin/PluginModule.java

```diff
@@ -246,7 +246,16 @@ protected void addEventProcessor(String name,
         registerJacksonSubtype(parametersClass, name);
     }
 
+    protected Multibinder<EventModifier> eventModifierBinder() {
+        return Multibinder.newSetBinder(binder(), EventModifier.class);
+    }
+
+    protected void addEventModifier(Class<? extends EventModifier> eventModifierClass) {
+        eventModifierBinder().addBinding().to(eventModifierClass);
+    }
+
     private MapBinder<String, EventStorageHandler.Factory> eventStorageHandlerBinder() {
+        Multibinder.newSetBinder(binder(), SchedulerCapabilities.class);
```

Comment:
was this supposed to be added with these changes?

Reply:
Oups, no. Good catch, thank you! I will fix it.